src/facilities/costCenters.spec.js

```diff
@@ -0,0 +1,540 @@
+        );
+      });
+
+      it('returns a fulfilled promise', function() {
+        return expect(promise).to.be.fulfilled;
+      });
+    });
+
+    context('when there is missing required information', function() {
+      it(`it throws an error when the cost center id is missing`, function() {
+        const expectedErrorMessage = `A cost center id is required for deleting a cost center.`;
+
+        const costCenters = new CostCenters(baseSdk, baseRequest, expectedHost);
+        const promise = costCenters.delete();
+
+        return expect(promise).to.be.rejectedWith(expectedErrorMessage);
+      });
+    });
+  });
+
+  describe('getAll', function() {
+    let expectedCostCenter;
+    let formatCostCenterFromServer;
+    let costCentersFromServer;
+    let promise;
+    let request;
+
+    beforeEach(function() {
+      const numberOfCostCenters = faker.random.number({
+        min: 1,
+        max: 10
+      });
+      expectedCostCenter = fixture.buildList('costCenter', numberOfCostCenters);
+      costCentersFromServer = fixture.buildList(
+        'costCenter',
+        numberOfCostCenters
+      );
+
+      formatCostCenterFromServer = this.sandbox
+        .stub(facilitiesUtils, 'formatCostCenterFromServer')
+        .callsFake((costCenter, index) => expectedCostCenter[index]);
+      request = {
+        ...baseRequest,
+        get: this.sandbox.stub().resolves(costCentersFromServer)
+      };
+
+      const costCenters = new CostCenters(baseSdk, request, expectedHost);
+      promise = costCenters.getAll();
+    });
+
+    it('gets a list of cost centers', function() {
+      expect(request.get).to.be.calledWith(`${expectedHost}/costcenters`);
+    });
+
+    it('formats the list of cost centers', function() {
+      return promise.then(() => {
+        expect(formatCostCenterFromServer).to.have.callCount(
+          costCentersFromServer.length
+        );
+        costCentersFromServer.forEach((costCenter) => {
+          expect(formatCostCenterFromServer).to.be.calledWith(costCenter);
+        });
+      });
+    });
+
+    it('returns a fulfilled promise with the cost centers', function() {
+      return expect(promise).to.be.fulfilled.and.to.eventually.deep.equal(
+        expectedCostCenter
+      );
+    });
+  });
+
+  describe('getAllByOrganizationId', function() {
+    context('when all required information is provided', function() {
+      let expectedCostCenter;
+      let expectedOrganizationId;
+      let formatCostCenterFromServer;
+      let costCentersFromServer;
+      let promise;
+      let request;
+
+      beforeEach(function() {
+        const numberOfCostCenters = faker.random.number({
+          min: 1,
+          max: 10
+        });
+        expectedCostCenter = fixture.buildList(
+          'costCenter',
+          numberOfCostCenters,
+          false
+        );
+        costCentersFromServer = fixture.buildList(
+          'costCenter',
+          numberOfCostCenters,
+          true
+        );
```

Comment:
I'm thinking this should be `costCentersFromServer = fixture.buildList('costCenter', numberOfCostCenters', null, { fromServer: true });`